packages/ai/src/ai/account/file_store.py

```
@@ -0,0 +1,425 @@
+    General-purpose binary file system interface built on top of IStore.
+
+    All paths are scoped to ``users/<client_id>/files/`` within the storage
+    backend, providing per-account isolation. All I/O is raw bytes.
+
+    Handle-based operations (open_write/read, write_chunk/read_chunk, close)
+    allow streaming large files without buffering everything in memory.
+    Each handle is tagged with a connection_id so it can be cleaned up when
+    the owning connection terminates.
+    """
+
+    def __init__(self, store: IStore, client_id: str):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)